.ci/rebuild_updated_recipes.py

```diff
@@ -16,12 +16,14 @@ def modified_recipes(branch='origin/master'):
     for file_path in git_diff:
         if 'recipes/' in file_path:
             recipe = file_path.split('/')[1]
-            recipes.add(recipe)
+            if recipe not in ["python.py", "hostpython.py"]:  # alias recipes
+                recipes.add(recipe)
```

Comment:
Actually now that I think of it, we're already using a set so this is dead code in a way